resources/usr/local/etc/sphinx/configs-available/standard.sphinx.conf

```diff
@@ -134,3 +134,194 @@ index vanilla_dev_Comment_Delta: vanilla_dev_Comment {
     source = vanilla_dev_Comment_Delta
     path = /usr/local/etc/sphinx/data/Comment_Delta
 }
+
+source vanilla_dev_KnowledgeArticle {
```

Comment:
I recommend moving this into a separate file, kind of like what we have for groups (e.g. [groups-standard.sphinx.conf](https://github.com/vanilla/vanilla-docker/blob/master/resources/usr/local/etc/sphinx/configs-available/groups-standard.sphinx.conf)). A Vanilla install without this addon is going to throw (non-fatal) errors during startup.

> ERROR: index 'vanilla_dev_KnowledgeArticleDeleted_Delta': sql_query_range: : range-query failed: Table 'vanilla_dev.GDN_articleRevision' doesn't exist (DSN=mysql://root:\*\*\*\@&#8203;database:3306/vanilla_dev).

Reply:
Check update please.